qgis_deployment_toolbelt/shortcuts/shortcut_freedesktop.template

```diff
@@ -17,5 +17,11 @@ Version=1.5
 X-MultipleArgs=false
 
 [Desktop Action ProfileFolder]
-Name=Ouvre le dossier du profil $profile_name
-Exec=open $profile_folder
+Name=Open $profile_name folder
+Exec=xdg-open $profile_folder
+Icon=folder
+
+[Desktop Action RemoveProfile]
+Name=Remove profile $profile_name from QGIS
+Exec=bash -c "zenity --question --title='Confirm QGIS profile removal' --text='Remove $profile_name from QGIS profiles?' && gio -r $profile_folder"
```

Comment:
I'm afraid this will only work on a GNOME based desktop because `zenity` and `gio` are not installed by default on a KDE desktop. 

Maybe you could try something like this (not tested):

```bash
"if [ '$XDG_CURRENT_DESKTOP' = 'KDE' ] || [ '$KDE_FULL_SESSION' = 'true' ]; then kdialog --yesno 'Remove $profile_name from QGIS profiles?' && rm -r $profile_folder; else zenity --question --title='Confirm QGIS profile removal' --text='Remove $profile_name from QGIS profiles?' && gio -r $profile_folder; fi"
```

Reply:
@ptitjano I've tested on fedora 42 with KDE https://fedoraproject.org/kde/ and zenity and gio are shipped.

Thanks for your oneliner but I'm not sure that it works in a .desktop without storing it in a bash file. I would like to avoid to ship a script and store it somewhere in the user folders.